src/test/java/seedu/sellsavvy/model/AddressBookTest.java

```diff
@@ -82,6 +82,37 @@ public void hasPerson_personWithSameIdentityFieldsInAddressBook_returnsTrue() {
         assertTrue(addressBook.hasPerson(editedAlice));
     }
 
+    @Test
+    public void hasSimilarPerson_noSimilar_returnsFalse() {
+        assertFalse(addressBook.hasPerson(ALICE));
```

Comment:
Shouldn't this be `addressBook.hasSimilarPerson`?

Reply:
in